README.md

```diff
@@ -272,10 +269,19 @@ func main() {
 
 ### Configuration
 
-The `APPOPTICS_TRACING_MODE` environment variable may be set to "always" or "never".
-- Mode "always" is the default, and will instruct AppOptics to consider sampling every inbound request for tracing.
-- Mode "never" will disable tracing, and will neither start nor continue traces.
-
+These environment variables may be set:
+
+| Variable Name        | Required           | Default  | Description |
+| -------------------- | ------------------ | -------- | ----------- |
+|APPOPTICS_SERVICE_KEY|Yes||The service key identifies the service being instrumented within your Organization. It should be in the form of ``<api token>:<service name>``.|
+|APPOPTICS_DEBUG_LEVEL|No|ERROR|Logging level to adjust the logging verbosity. Increase the logging verbosity to one of the debug levels to get more detailed information added to the app logs. Possible values: DEBUG, INFO, WARN, ERROR|
```

Comment:
Minor, can shorten to `Increase the logging verbosity to get more detailed information...`